packages/expo/src/launch/withExpoRoot.web.tsx

```diff
@@ -14,15 +13,7 @@ export default function withExpoRoot<P extends InitialProps>(
         exp: { ...this.props.exp, errorRecovery: ErrorRecovery.recoveredProps },
```

Comment:
I'm not sure if we still need these ErrorRecovery props, not sure what they do or where they're used.

Reply:
from this thing https://docs.expo.io/versions/latest/sdk/error-recovery/

> If your project crashes in production as a result of a fatal JS error, Expo will reload your project. If you've set these props, they'll be passed to your reloaded project's initial props under exp.errorRecovery